pyperf/_linux_memory.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
         threading.Thread.__init__(self)
         self.peak_usage = 0
         self._done = threading.Event()
-        self.sleep = 0.001   # 1 ms
+        self.sleep = 0.01   # 10 ms
```

Comment:
I like to write 3 digits to align to milliseconds :-p it's just a personal preference.

```suggestion
        self.sleep = 0.010   # 10 ms
```